### ISSUE TITLE
[Feature Request] Add `<autogenerated />` comment to generated file header

### ISSUE THREAD

**Issue by filzrev** [aelij/IgnoresAccessChecksToGenerator#19]
When some code analyzer rules are enabled. (e.g. IDE0073: Require file header)
Warnings are reported on generated `IgnoresAccessChecksTo.cs` file.

It may be better to add the following comment to the `IgnoresAccessChecksTo.cs` header to suppress code analyzer related warnings.

```
// <auto-generated />
```

**Comment by aelij** [comment]
Fixed in #24

**Comment by koder-man** [comment]
Hi! When will this version be available on Nuget?

**Comment by aelij** [comment]
Published, thanks
